include/mxnet/c_api.h

```diff
@@ -71,8 +67,8 @@ struct NativeOpInfo {
 };
 
 struct NDArrayOpInfo {
-  bool (*forward)(int, mxnet::NDArray**, int*, void*);
-  bool (*backward)(int, mxnet::NDArray**, int*, void*);
+  bool (*forward)(int, void**, int*, void*);
```

Comment:
You can use NDArrayHandle for more readiblity. NDArrayHandle is typedefed as void*